### PR TITLE
UIP-573: remove material completely from the codebase

### PR DESCRIPTION
**Pull request by rey-wright**
## Description
My attempt to delete materialPalette and all associated documentation from the codebase

## Screenshots
N/A

## Checklist
- [x] Docs have been rebuilt (`yarn build:full`)
- [x] All unit tests pass
- [x] Snapshots have been updated
- [x] No lint errors or warnings have been introduced
- [x] **[Version bumped](https://github.com/cbinsights/form-design-system#updating-version-number) if appropriate**

### CODE REVIEW

**Comment by akdetrick** [comment]
Applications that consume FDS will likely rely on this in their theme file:
https://unpkg.com/browse/@cbinsights/fds@6.13.0-beta.c8cd6c3/lib/dictionary/js/materialPalette.js

Would the migration plan be to hard-code some of these colors in the theme files?

**Comment by akdetrick** [comment]
🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 🔪 
Screen Shot 2020-01-22 at 6 30 48 PM
🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡 🗡

File: scripts/node/dictionary/transforms.js [hunk 1]
```diff
@@ -98,43 +98,6 @@ const toRgbComponents = (prop) => {
...
-
-  const flattenReducer = (acc, curr) => {
-    acc[curr.key] = curr.color;
-    return acc;
-  };
-
-  const tints = tintKeys
-    .map((key, i) => ({
-      key,
-      color: materialTint(prop.value, i + 1),
```
Comment (akdetrick) [inline scripts/node/dictionary/transforms.js]: now that these material color functions are no longer used, you can probably get rid of a bunch of stuff in `util/color.js`

**Comment by rey-wright** [comment]
> Would the migration plan be to hard-code some of these colors in the theme files?

Yep. I already did this for `cbi-site` and `cbi-tables`. The only thing not migrated is admin.

**Pull request merged**